### DISCUSSION TITLE
Archive formats support?

### DISCUSSION THREAD

**Discussion by HoreaM** [manyfold3d/manyfold#3126] [Support]
According to the following list, archives formats should be supported in Manyfold:
https://manyfold.app/manual/supported_formats.html

Despite this fact, I can't see or download any archives through the Manyfold UI, am I doing something wrong?

**Comment by Floppy** [comment]
Sorry for the delay, I didn't spot this question somehow! Archives are supported for uploading, but I don't think are indexed for display and download. Though thinking about it, I thought I'd seen it display .zip files recently... which is odd. 🤔 

Anyway, am I right in thinking that you'd like to have ZIP (or other archive) files as a file within a model? There are some feature requests open for things in that area, such as #456, but it's useful to know what people want so we can add it in.

**Reply by HoreaM** [reply]
`Anyway, am I right in thinking that you'd like to have ZIP (or other archive) files as a file within a model?`
Yes, that is correct, this feature would be extremely useful for me!
Thanks a lot for getting back to me.

**Reply by HoreaM** [reply]
Is this something you are thinking of adding at some point? I forgot to ask, sorry!

**Reply by Floppy** [reply]
I forgot to come back on this, sorry! ZIP files should appear in models if you upload them; I definitely have got a few that had a zip-inside-a-zip when I added them, and the inner zipfile stayed around. So it should work...

**Reply by HoreaM** [reply]
Thanks for getting back to me! 
So this does not seem to be the case for me, my entire library consists almost exclusively of models that contain only ZIPs and pictures in their folder. The pictures are recognized, but the ZIP files never show up in the model page in Manyfold.
One more detail that maybe be of relevance is that I don't really upload the models through the interface, I always add them in the folder and then do a manual scan for changes to add the models in Manyfold.

**Reply by Floppy** [reply]
ah, that could be it, perhaps it's not scanning them on disk, but is when uploading. I'll check!

**Reply by Floppy** [reply]
I've put it on my todo list at #3392, so I won't forget again!

**Reply by HoreaM** [reply]
Thank you, I really appreciate it!
